gulpfile.js

```diff
@@ -54,17 +53,13 @@ function install(platformInfo, packageJSON) {
     const packageManager = new PackageManager(platformInfo, packageJSON);
     const logger = new Logger(message => process.stdout.write(message));
     const debuggerUtil = new debugUtil.CoreClrDebugUtil(path.resolve('.'), logger);
-    const debugInstaller = new debugInstall.DebugInstaller(debuggerUtil);
 
     return packageManager.DownloadPackages(logger)
         .then(() => {
             return packageManager.InstallPackages(logger);
         })
         .then(() => {
             return util.touchInstallFile(util.InstallFileType.Lock)
-        })
-        .then(() => {
-            return debugInstaller.finishInstall();
```

Comment:
Do we still need to write the install complete file?

Reply:
Yes. We should still need it.